pkgs/development/python-modules/reportlab/default.nix

```diff
@@ -23,12 +23,17 @@ in buildPythonPackage rec {
   buildInputs = [ ft pillow ];
 
   postPatch = ''
-    rm tests/test_graphics_barcode.py
+    # Remove all the test files that require access to the internet to pass.
+    rm tests/test_lib_utils.py
+    rm tests/test_platypus_general.py
+
+    # Remove the tests that require Vera fonts installed
```

Comment:
@dotlambda I can probably figure out how to get Vera font for the test, (I tried adding `pkgs.ttf-bitstream-vera` to the `checkInputs` but it did not work).

@dotlambda PTAL.

Reply:
I don't think that's necessary. Thanks a lot for your work!